pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/BatchConfig.java

```diff
@@ -32,6 +32,7 @@ public class BatchConfig {
 
   private final FileFormat _inputFormat;
   private final String _inputDirURI;
+  private final boolean _searchRecursively;
```

Comment:
Is this used? If so, the default for it should also be `true`

Reply:
Done